home.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="./assets/styles/homeStyle.css">
+</head>
+<body>
+    <head>
+        <div class="main-container">
+        <div class="mobile-nav">
+            <div class="moblile-menu-icon">
+                <!-- <a id="mobile-icon-btn" href="#"> <img src="./assets/images/mobile_menu.png" alt="menu hamburger icon"></a> -->
+            </div>
+            <div id="myNav" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <div class="overlay-content">
+                    <a href="./About.html" onclick="closeNav()">About</a>
+                    <a href="#about-social" onclick="closeNav()">Program</a>
+                    <a href="#contact" onclick="closeNav()">Join</a>
+                    <a href="#News" onclick="closeNav()">News</a>
+                    <a href="#contact" onclick="closeNav()">CC Campaign</a>
+                </div>
+            </div>
+            <div class="desk-nav">
+                <div class="logo-png">
+                    <a href="#"> <img src="./assets/images/logopng.png" alt="logo image"> </a>
+                </div>
+                <div class="desk-nav-items">
+                    <ul>
+                        <li class="menu-item"><a href="./About.html"> About </a> </li>
+                        <li class="menu-item">Progam</li>
+                        <li class="menu-item">Join</li>
+                        <li class="menu-item">Sponsor</li>
+                        <li class="menu-item">News</li>
+                    </ul>
+                </div>
+            </div>
+            <nav class="flex-nav"><button onclick="openNav()">☰</button></nav>
```

Comment:
- [ ] Kindly ensure that your hamburger button matches the design guidelines given so that your website can look better 😄 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190513265-8e3000fc-bff6-49f0-b3fb-ea218b513743.png)  | ![image](https://user-images.githubusercontent.com/63473584/190513343-e42c8204-8807-4682-8a46-b1a9f7958a4a.png) |